EventFilter/EcalTBRawToDigi/src/CamacTBDataFormatter.cc

```diff
@@ -127,22 +127,17 @@ void CamacTBDataFormatter::interpretRawData( const FEDRawData & fedData,
   LogDebug("CamacTBDataFormatter") << "\n\nword:\t" << a;
   
   b = (a& 0xff000000);
-  b = b >> 24;
```

Comment:
Lines like this can't be deleted. It is changing `b`, which is used in the very next line. Please restore all these lines that have an effect.